python/samples/agentchat_graphrag/README.md

```diff
@@ -0,0 +1,58 @@
+# Building a Multi-Agent Application with AutoGen and GraphRAG
+
+In this sample, we will build a chat interface that interacts with a `RoundRobinGroupChat` team built using the [AutoGen AgentChat](https://microsoft.github.io/autogen/dev/user-guide/agentchat-user-guide/index.html) API and the GraphRAG framework.
+
+
+## High-Level Description
+
+The `app.py` script sets up a chat interface that communicates with the AutoGen team. When a chat starts, it:
+
+- Initializes an AgentChat team with both local and global search tools.
+- As user query is sent to the team with the agent, which must select the appropriate tool to use, the query is then passed to the appropriate tool to respond.
+- As agents respond/act, their responses are streamed back to the chat interface.
+
+## What is GraphRAG?
+
+GraphRAG (Graph-based Retrieval-Augmented Generation) is a framework designed to enhance multi-agent systems by providing robust tools for information retrieval and reasoning. It leverages graph structures to organize and query data efficiently, enabling both global and local search capabilities.
+
+Global Search: Global search involves querying the entire indexed dataset to retrieve relevant information. It is ideal for broad queries where the required information might be scattered across multiple documents or nodes in the graph.
+
+Local Search: Local search focuses on a specific subset of the data, such as a particular node or neighborhood in the graph. This approach is used for queries that are contextually tied to a specific segment of the data.
+
+By combining these search strategies, GraphRAG ensures comprehensive and context-sensitive responses from the multi-agent team.
+
+## Setup
+
+To set up the project, follow these steps:
+
+1. Download the plain text version of "The Adventures of Sherlock Holmes" from [Project Gutenberg](https://www.gutenberg.org/ebooks/1661) and save it to `data/input/sherlock_book.txt`.
+
+2. Adjust the `settings.yaml` file with your LLM and embedding configuration. Ensure that the API keys and other necessary details are correctly set.
+
+3. Replace the LLM client and credentials in `app.py` as needed to match your setup.
```

Comment:
We don't need to modify app.py now because we are using model config loader. So let's update this instruction to how to setup the `model_config.yml` file in the current directory.